Pipfile.lock

```diff
@@ -795,14 +817,6 @@
             ],
             "version": "==2.3.3"
         },
-        "atomicwrites": {
```

Comment:
Why the removal of Atomicwrites?

Reply:
I did not manually modify this file, so I am confused how it was included in the first place, to be honest. I created this branch off of the current state of master and the normal pipfile does not include any of these tools as far as I can tell.

Reply:
Reverted my changes to Pipfile.lock to prevent crashes on non-linux or mac operating systems